demo/admin/src/dam/ImportFromUnsplash.tsx

```diff
@@ -1,6 +1,7 @@
 import { useApolloClient } from "@apollo/client";
 import { CancelButton, messages, SaveButton } from "@comet/admin";
 import { useDamAcceptedMimeTypes } from "@comet/cms-admin";
+import { useFolderContext } from "@comet/cms-admin/lib/dam/DamTable";
```

Comment:
Again, a private API 😁 Please rename to `useCurrentDamFolder` and add to public API